Kickstarter-iOS/AppDelegate.swift

```diff
@@ -144,22 +147,28 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
       .observeForUI()
       .observeValues(UIApplication.shared.unregisterForRemoteNotifications)
 
-    self.viewModel.outputs.configureHockey
+    self.viewModel.outputs.configureAppCenterWithData
       .observeForUI()
-      .observeValues { [weak self] data in
-        guard let _self = self else { return }
-        let manager = BITHockeyManager.shared()
-        manager.delegate = _self
-        manager.configure(withIdentifier: data.appIdentifier)
-        manager.crashManager.crashManagerStatus = .disabled
-        manager.isUpdateManagerDisabled = data.disableUpdates
-        manager.userID = data.userId
-        manager.userName = data.userName
-        manager.start()
-        manager.authenticator.authenticateInstallation()
+      .observeValues { data in
+        let customProperties = MSCustomProperties()
+        customProperties.setString(data.userName, forKey: "userName")
+
+        MSAppCenter.setUserId(data.userId)
+        MSAppCenter.setCustomProperties(customProperties)
+
+        MSCrashes.setDelegate(self)
+
+        MSAppCenter.start(
+          data.appSecret,
+          withServices: [
+            MSAnalytics.self,
+            MSCrashes.self,
+            MSDistribute.self
```

Comment:
Need to confirm that `MSDistribute` works correctly for us. In the set up [docs](https://docs.microsoft.com/en-us/appcenter/sdk/distribute/ios) it mentions adding the app secret to `Info.plist` which is obviously problematic for us. So I didn't do it but I'm not sure if it will cause issues.